### DISCUSSION TITLE
Introduce the version with the App Cast Generator

### DISCUSSION THREAD

**Discussion by anxofcostas** [NetSparkleUpdater/NetSparkle#389] [Q&A]
Hello!

I´ve started to use the App Cast Generator but idk how to assign a version to my project. How can I do it? I think it would be a simple solution but I don´t know how to do it.

Thank yoou!

**Comment by Deadpikle** [comment]
Hi @anxofcostas,

By default, it tries to extract the version from the update file itself, but depending on your setup, this may not always work. You can put the version information in your file name. Use the `-f`/`--file-extract-version`  flag with `-f true` and then name your files something like `My App 1.4.2.exe` or similar.

**Reply by anxofcostas** [reply]
Hello @Deadpikle !

I have 2 questions:

Do you know how can I remove the spaces on my URL and path fields in the XML file?

How can I write the whole path in the URL field of a local file when I going to create the XML file?
Thank you do much!

**Reply by Deadpikle** [reply]
Hi,

I'm not sure what you're asking about removing spaces in your URL and path fields. Can you give an example?

Local file paths may or may not be working right in the current version of the app cast generator. Enhancements and PR's are welcome if that is needed before I can get around to it, which might be a long time from now.
